app/settings/account/-components/request-deactivation/component.ts

```diff
@@ -0,0 +1,81 @@
+import { tagName } from '@ember-decorators/component';
+import { action } from '@ember-decorators/object';
+import { alias } from '@ember-decorators/object/computed';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import { task } from 'ember-concurrency';
+import config from 'ember-get-config';
+import I18N from 'ember-i18n/services/i18n';
+import Toast from 'ember-toastr/services/toast';
+
+import User from 'ember-osf-web/models/user';
+import UserSettingModel from 'ember-osf-web/models/user-setting';
+import CurrentUser from 'ember-osf-web/services/current-user';
+
+@tagName('')
+export default class DeactivationPane extends Component.extend({
+    loadSettings: task(function *(this: DeactivationPane) {
+        const { user } = this.currentUser;
+
+        if (!user) {
+            return undefined;
+        }
+        this.settings = yield user.belongsTo('settings').reload();
+    }),
+
+    saveSettings: task(function *(this: DeactivationPane, successMessage: string) {
+        try {
+            if (this.settings !== undefined) {
+                yield this.settings.save();
```

Comment:
Don't need to save because vmf already does.

Reply:
I forgot: This does not use ValidatedModelForm, so this is actually a useful save.